server/bin/gen_fake_report.py

```diff
@@ -38,16 +58,19 @@ class VirtWhoReportGenerator(AbstractVirtWhoReportGenerator):
     def _generate(self):
         report = {}
         for x in range(0, self.hypervisors):
-            report[str(uuid.uuid4())] = [self.gen_guest() for y in
-                range(0, self.guests)]
+            report[str(uuid.uuid4())] = [
+                self.gen_guest() for y in range(0, self.guests)
+            ]
         return report
 
 
 class AsyncVirtWhoReportGenerator(AbstractVirtWhoReportGenerator):
 
     def gen_hypervisor(self, num_guests, min_guests=0, max_guests=None):
-        return {"uuid": str(uuid.uuid4()),
-                "guests": [self.gen_guest() for x in range(0, num_guests)]}
+        return {"hypervisorId": {"hypervisorId": str(uuid.uuid4())},
+                "name": "thinkpad-" + gen_random_string(),
```

Comment:
This will function, but I'd like this to be more generic so that consumer names are indicative of what they are. Something like "hypervisor-" or "generated_hypervisor-", etc. would be better for sifting through test data and logs once it hits Candlepin.

Reply:
Changed it to use ```generated_hypervisor``` as prefix.